app/helpers/truncate-text.js

```diff
@@ -0,0 +1,19 @@
+import {helper} from '@ember/component/helper';
+
+function truncateText(params, hash) {
```

Comment:
Is this function being used anywhere? I can't find any usages.

Reply:
Used in template.

Reply:
Whoops, okay!